src/typing_extensions.py

```diff
@@ -2986,7 +3007,8 @@ def is_protocol(__tp: type) -> bool:
         return (
             isinstance(__tp, type)
             and getattr(__tp, '_is_protocol', False)
-            and __tp != Protocol
+            and __tp is not Protocol
+            and __tp is not getattr(typing, "Protocol", object())
```

Comment:
`is_protocol(typing.Protocol)` was evaluating to `True` on PyPy-3.8. I don't know why that is, really, but this change fixes it.

Reply:
I think I used `!=` on purpose here so that Protocol can have a custom `__eq__` that helps us out. What you're doing here should work too, though.

Reply:
Yeah, I have no idea why `_ProtocolMeta.__eq__` doesn't seem to be doing what it should be doing on PyPy 3.8. I suspect another bug in PyPy. Easy enough for us to work around it here, though :)